package-lock.json

```diff
@@ -2777,6 +2777,20 @@
 			"integrity": "sha1-GwqzvVU7Kg1jmdKcDj6gslIHgyc=",
 			"dev": true
 		},
+		"fuzzysearch-js": {
```

Comment:
not required anymore? I think we can revert this change right?

Reply:
Good spotting :eyeglasses: